projects/packages/plans/src/class-current-plan.php

```diff
@@ -364,4 +380,11 @@ function_exists( 'wpcom_feature_exists' ) &&
 
 		return false;
 	}
+
+	/**
+	 * Refreshess the plan data.
+	 */
+	public static function refresh_plan_data() {
```

Comment:
Shall we just hook into `refresh_from_wpcom` instead? We could rename that if `refresh_plan_data` is more appropriate.